polus-recycle-vector-plugin/README.md

```diff
@@ -1,21 +1,30 @@
+This WIPP plugin applies an existing stitching vector to an image collection.
+The
+[WIPP image assembling plugin](https://github.com/usnistgov/WIPP-image-assembling-plugin)
+and the
+[WIPP pyramid building plugin](https://github.com/usnistgov/WIPP-pyramid-plugin)
+do not have a way to apply a stitching vector to new sets of images. For
+example, it is currently possible to stitch the first channel of a multi-channel
+image and apply the same stitching vector to the other channels. This plugin
+creates a new stitching vectors that will an existing single stitching vector to
+a different set of images.
+
+This plugin uses 
+[filepatterns](https://filepattern.readthedocs.io/en/latest/),
+which is a variant of regular expressions similar to what MIST uses. See
+**Input Regular Expressions** for more information.
 
 ## Input Regular Expressions
-This plugin uses [filename patterns](https://github.com/USNISTGOV/MIST/wiki/User-Guide#input-parameters) similar to that of what MIST uses. In particular, defining a filename variable is surrounded by `{}`, and the variable name and number of spaces dedicated to the variable are denoted by repeated characters for the variable. For example, if all filenames follow the structure `filename_TTT.ome.tif`, where TTT indicates the timepoint the image was captured at, then the filename pattern would be `filename_{ttt}.ome.tif`.
-
-All filename patterns must include `x` and `y` grid positions for each image or a sequential position `p`, but not both. This differs from MIST in that `r` and `c` are used to indicate grid row and column rather than `y` and `x` respectively. This change was made to remain consistent with Bioformats dimension names and to permit use of `c` as a channel variable.
-
-In addition to the position variables (both `x` and `y`, or `p`), the only other variables that can be used are `r`, `z`, `c`, and `t`.
-
-**Version 1.1 Update:** If no inputs are provided in the `groupBy` variable, then all non-positional variables that vary from the stitching vector will cause a new stitching vector to be generated. For example, if replicate, x, and y are all contained within a stitching vector but the vector contains only files for channel 1 (e.g. r{rrr}_x{xxx}_y{yyy}_c001.ome.tif), then the stitching vector filepattern can be set to r{rrr}_x{xxx}_y{yyy}_c{ccc}.ome.tif and `--groupBy r` can be used to recycle the stitching vector for additional channels.
+A `filepattern` is a way of indicating where variables in a file name are
+located. A `filepattern` defines a filename variable using `{}`, and the
+variable name and number of spaces dedicated to the variable are denoted by
+repeated characters for the variable. For example, if all filenames follow the
+structure `filename_TTT.ome.tif`, where TTT indicates the timepoint the image
+was captured at, then the filename pattern would be `filename_{ttt}.ome.tif`.
+For more information on `filepattern`, see the 
```

Comment:
filename pattern could be `filename_{ttt}.ome.tif` or `filename_{t+}.ome.tif`. Just a suggestion so that users are familiar with the `+` notation as well.